### ISSUE TITLE
code about validate

### ISSUE THREAD

**Issue by tosvqo** [ZPDu/Redesigning-Multi-Scale-Neural-Network-for-Crowd-Counting#5]
作者您好，请问一下，在validate阶段，如果pred_map=net(img)[0]，在计算loss = mseloss(pred_map.squeeze(1), gt_map)会报错，我的理解是因为pred_map其实是包含7个密度图输出的列表，所以改为了pred_map=net(img)[0][-1]，将最终的密度图输出Eout(代码中对应density)来计算validate的MSELOSS，但是即使这样修改后，不加relloss的SHA的MAE也只有62.，加上relloss也只有60.[best] [mae 60.4 mse 92.3], [val loss 18699.08538953], [epoch 175]。这是为什么？
